src/js/frontend.js

```diff
@@ -0,0 +1,6 @@
+/**
+ * Register sw.js
+ */
+if ('serviceWorker' in navigator) {
+  navigator.serviceWorker.register('/sw.min.js', {scope: './'});
```

Comment:
- what happens if service worker is registered in browser under `/sw.min.js` and wordpress plugin is installed on non apache server, let's say nginx ? I mean if server is not ready to serve the service worker file, we shouldn't register it in browser, because it'll cause some errors.